nodemon.json

```diff
@@ -10,5 +10,6 @@
         "template.env",
         "TODO"
     ],
-    "verbose": true
+    "verbose": true,
+    "exec": "npm run start"
```

Comment:
should this be `npm run dev`?

Reply:
It should not,  no. `npm run start` is what actually triggers the running of the application, but nodemon allows for hotswapping code and on-the-fly recompilation, so `npm run dev` calls nodemon which then boots up the actual ts-node execution.